app/views/layouts/application.haml

```diff
@@ -22,10 +22,10 @@
     %meta{name: "application-name", content: "Glowfic Constellation"}
     %meta{name: "msapplication-config", content: asset_path('favicons/browserconfig.xml')}
     %meta{name: "theme-color", content: "#ffffff"}
-    = favicon_link_tag asset_path('favicons/favicon.ico')
-    = favicon_link_tag asset_path('favicons/favicon-32x32.png'), type: 'image/png', rel: 'icon', sizes: '32x32'
-    = favicon_link_tag asset_path('favicons/favicon-16x16.png'), type: 'image/png', rel: 'icon', sizes: '16x16'
-    = favicon_link_tag asset_path('favicons/apple-touch-icon.png'), type: 'image/png', rel: 'apple-touch-icon', sizes: '180x180'
+    = favicon_link_tag 'favicons/favicon.ico'
```

Comment:
[Here](http://guides.rubyonrails.org/v4.1/asset_pipeline.html#coding-links-to-assets) seems to not expect `asset_path` to be used despite sprockets being involved.